code/style.css

```diff
@@ -1,4 +1,755 @@
+
+  header {
+    flex-direction: row;
+    padding: 10px 40px 10px 40px;
+  }
+
+  nav {
+    display: none;
+  }
+  
+  nav.tablet {
+    display: flex;
+    width: 50%;
+    justify-content: flex-end;  
+  }
+
+  nav.tablet a {
+    font-size: 22px;
+    font-family: 'Open Sans Condensed', sans-serif, Arial, Helvetica;
+    color: white;
+    list-style-type: none;
+    text-decoration: none;
+    margin-left: 30px;
+  }
+}
+
+/* For computer only */
+@media screen and (min-width: 1025px) {
+
+  header {
+    flex-direction: row;
+    padding: 10px 40px 10px 40px;
+  }
+
+  nav {
+    width: 50%;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  nav a {
+    font-size: 14px;
+    font-family: 'Open Sans Condensed', sans-serif, Arial, Helvetica;
+    color: white;
+    list-style-type: none;
+    text-decoration: none;
+    margin-left: 30px;
+  }
+
+  nav a:hover {
+    text-decoration: underline;
+  }
+
+  /* Remove tablet navbar from computer */
+  nav.tablet {
+    display: none;
+  } 
+}
+
+/* NEWSLETTER SECTION */
+
+/* Remove newsletter section from mobile */
+@media screen and (max-width: 667px) {
+  section.newsletter {
+    display: none;
+  }
+
+  h2.newsletter {
+    display: none;
+  }
+
+  button.newsletter {
+    display: none;
+  }
+
+  button.newsletter-tablet {
+    display:none;
+  }
+}
+
+/* For tablet only */
+@media screen and (min-width: 667px) and (max-width: 1024px) {
+
+  section.newsletter {
+    background-image: url("img/newsletter-background.jpg");
+    height:200px;
+    background-repeat:no-repeat;
+    background-size:auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 85px;
+    width: 100%;
+  }
+
+  h2.newsletter {
+    display:none;
+  }
+
+  button.newsletter {
+    display:none;
+  }
+
+  button.newsletter-tablet {
+    background-color: black;
+    border: none;
+    color: white;
+    padding: 14px 22px;
+    margin: 16px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 26px;
+    font-family: 'Open Sans Condensed', sans-serif, Arial, Helvetica;
+    cursor: pointer;
+    box-shadow: 1px 1px 2px -1px rgba(46,46,46,1);
+  }
+}
+
+/* For computer only */
+@media (min-width: 1025px) {
+
+  button.newsletter-tablet {
+    display:none;
+  }
+
+  section.newsletter {
+    background-image: url("img/newsletter-background.jpg");
+    height:250px;
+    background-repeat:no-repeat;
+    background-size:auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 85px;
+    width: 100%;
+  }
+  
+  h2.newsletter {
+    display: flex;
+    font-size: 22px;
+    font-family:'Cormorant Garamond', serif, Georgia;
+    font-style: italic;
+    font-weight: lighter;
+    position: relative;
+    animation-name: newsletter;
+    animation-duration: 4s;
+  }
+
+  /* Animations for h2.newsletter */
+  @keyframes newsletter {
+    0% {right:100%;}
+    100% {right:0px;}
+  }
```

Comment:
Love the animation, great job 👏 

When it comes to the one-liners, try to write it like this instead. 
```
  @keyframes newsletter {
    0% {
     right:100%;
     }
    100% {
     right:0px;
     }
  }
```